backend.native/compiler/ir/backend.native/src/org/jetbrains/kotlin/backend/konan/ObjCInterop.kt

```diff
@@ -34,8 +34,11 @@ import org.jetbrains.kotlin.utils.addToStdlib.firstNotNullResult
 
 internal val interopPackageName = InteropFqNames.packageName
 internal val objCObjectFqName = interopPackageName.child(Name.identifier("ObjCObject"))
+internal val objCObjectIdSignature = getPublicSignature(interopPackageName, "ObjCObject")
```

Comment:
Can we avoid duplication here by implementing a function like `getTopLevelPublicSignature(fqName: FqName)`